### DISCUSSION TITLE
Simple "check-box" recurring donations?

### DISCUSSION THREAD

**Discussion by rgnicholas760** [jonas-hoebenreich/wc-donation-platform#48] [Q&A] [answered]
Hello... I like this donation plugin and appreciate the work and care in helping us, nonprofits, to use Woocommerce to receive donations. Anyway, I am trying to figure out how - or if it is possible - to simply take a form that I have created with multiple donation choices and have a simple check-box to make the end result be recurring?

This is the form in question:  https://www.cefjacksontn.com/donate/

I see this in other systems, like Charitable for instance.... but I am trying to keep my cost down as much as possible. The native Woocommerce subscriptions plugin might possibly do this, but I am looking for cheaper alternatives. PLEASE share your ideas or a way to make my form have this functionality. A bonus would be for the records / transactions to show up in the Woopayments area, making it simpler to manage all donations.

**Comment by jonas-hoebenreich** [comment, marked as answer]
Hi rgnicholas760,

thanks for reaching out.
This functionality requires custom code. You can follow this guide on how to set it up:
Documentation: https://wcdp.jonh.eu/documentation/advanced/advanced-layout-expert-custom-code-with-action/
Demo: https://wcdp.jonh.eu/demos/custom-html-style-3/
This demo requires WooCommerce Subscriptions (https://wcdp.jonh.eu/documentation/integrations/recurring-donations-with-woocommerce-subscriptions/)

Best,
Jonas
